translate/cloud-client/src/test/java/com/example/translate/QuickstartSampleIT.java

```diff
@@ -35,6 +37,19 @@
   private ByteArrayOutputStream bout;
   private PrintStream out;
 
+  private static void requireEnvVar(String varName) {
+    assertNotNull(
+            System.getenv(varName),
```

Comment:
We realized this in another PR, but we've been doing this backwards. It's `assertNotNull(Message, ObjectToAssert)`.